docs/deployment/feature_gates.md

```diff
@@ -29,9 +29,11 @@ The following tables are a summary of the feature gates that you can set on diff
 | UseNamespacedCloudProfile | `false` | `Alpha` | `1.92`  |         |
 | ShootManagedIssuer        | `false` | `Alpha` | `1.93`  |         |
 | VPAForETCD                | `false` | `Alpha` | `1.94`  | `1.96`  |
-| VPAForETCD                | `true`  | `Beta`  | `1.97`  |         |
+| VPAForETCD                | `true`  | `Beta`  | `1.97`  | `1.104` |
+| VPAForETCD                | `true`  | `GA`    | `1.105` |         |
 | VPAAndHPAForAPIServer     | `false` | `Alpha` | `1.95`  | `1.100` |
-| VPAAndHPAForAPIServer     | `true`  | `Beta`  | `1.101` |         |
+| VPAAndHPAForAPIServer     | `true`  | `Beta`  | `1.101` | `1.104` |
+| VPAAndHPAForAPIServer     | `true`  | `GA`    | `1.105` |         |
```

Comment:
Nit: Move them to the table below as they are now graduated feature gates.

Reply:
I pushed 0f1da3988938a00c4c7c77d6ae2f7214eed3a92d to address the comment.

Reply:
I also thought that, but saw that we did not do it like that for any previous PRs that move feature gates to GA.